SLRemoting/SLObject.m

```diff
@@ -229,7 +229,7 @@ + (NSDictionary *)convertArguments:(NSDictionary *)prop {
         id value = [prop objectForKey:key];
         if ([value isKindOfClass:[NSDate class]]) {
             NSString *jsonString = [value convertToJSONString];
-            value = @{ @"$type": @"date", @"$data": jsonString };
+            value = jsonString ;
```

Comment:
>Is this perhaps a side-effect of my coercion refactor that I did not discover?

There was some coercion issue, don't remember ATM. Will update you with the findings.